cli/cmd/plugin/standalone-cluster/log/log.go

```diff
@@ -243,7 +237,7 @@ func (l *CMDLogger) Style(indent int, color string) Logger {
 // unquoteCodePoint takes the unicode value of a symbol and makes it usable for printing.
 func unquoteCodePoint(s string) string {
 	r, _ := strconv.ParseInt(strings.TrimPrefix(s, "\\U"), 16, 32)
-	return string(r)
+	return fmt.Sprintf("%-2s", string(r))
```

Comment:
This approach won't work because emojis have variable width size. Some taking up 2 columns vs the standard 1.

See the spacing inconsistency:

![image](https://user-images.githubusercontent.com/6200057/139702030-67cd620e-9b6f-486e-947a-b8a203268deb.png)

Reply:
Yeah, it's the same issue we have now where we put in spaces and have to hope it formats OK. Maybe we should vet our emoji selection to make sure they are consistent?

Reply:
+1 for vetting emoji selection. I'm also curious how kind does this since they seem to use emojis of consistent width throughout their output

Reply:
Sounds good, can we do that and make changes the logging package in a diff PR?

Also, is there a clever way to determine the width?

Reply:
> how kind does this since they seem to use emojis of consistent width throughout their output

I think kind prints emojis at the end of the line? Which means you probably don't notice this.

Reply:
> I think kind prints emojis at the end of the line? Which means you probably don't notice this.

Ah! Good way to get around this 😂

I can take a look at the emoji / unicode stuff and see if there's a good way to approach this

Reply:
Yep, was just about to remove that part from this PR since it's really a different concern than the main point of this one. We can figure that out later.

I tried looking at rune width, but they all are the same. And play.golang.org prints them out with a consistent width, so I think this has more to do with the terminal than having a variable data width.

If we care stongly enough, we could maybe use a tab writer to force more consistency, but not sure if we could make the width small enough for that not to look weird.

Something to investigate later, at any rate.

Reply:
The 2 column emojis were handled in: https://github.com/vmware-tanzu/community-edition/pull/2420

Reply:
Yep, this was removed in the latest update. Looks much better in 2420.